CMakeLists.txt

```diff
@@ -137,7 +143,9 @@ if (ENABLE_SANITIZERS)
 endif()
 
 # adoc for man generation
-include(AdocMan)
+if (ENABLE_DOC)
```

Comment:
Will this work correctly if ENABLE_DOC is not upper-case, i.e. was set like -DENABLE_DOC=on or -DENABLE_DOC=off ?

Reply:
Yes, should work correctly


>    True if the constant is 1, ON, YES, TRUE, Y, or a non-zero number. False if the constant is 0, OFF, NO, FALSE, N, IGNORE, NOTFOUND, the empty string, or ends in the suffix -NOTFOUND. **Named boolean constants are case-insensitive**. If the argument is not one of these specific constants, it is treated as a variable or string and the following signature is used.